src/site/content/en/blog/preload-optional-fonts/index.md

```diff
@@ -0,0 +1,108 @@
+---
+title: 'No layout shifting and FOIT with font-display: optional'
+subhead: 'Starting in Chrome 82, link rel="preload" and font-display: optional can be combined to remove layout jank completely'
+authors:
+  - houssein
+date: 2020-03-18
+hero: hero.jpg
+description: |
+  This post covers new optional font loading behavior with preloading in Chrome 82
```

Comment:
Writers frequently do something more like this for the description (taking from your opening paragraph:

By optimizing rendering cycles, Chrome 82 eliminates layout shifting when
preloading optional fonts. Combining `<link rel="preload">` with `font-display: optional` is the
most effective way to guarantee jank-free rendering of custom fonts.